src/drivers/cmakeServerDriver.ts

```diff
@@ -19,6 +17,8 @@ import { errorToString } from '@cmt/util';
 import * as nls from 'vscode-nls';
 import * as ext from '@cmt/extension';
 import { BuildPreset, ConfigurePreset, TestPreset } from '@cmt/preset';
+import { ServerCodeModelContent } from '@cmt/drivers/cmakeServerClient';
```

Comment:
There is still an import `cms` for `cmakeServerClient` (line 9).  Do you want to use that, or import each type in separately?  We shouldn't have both

Reply:
removed the second one.